### ISSUE TITLE
Text drawing broken in develop DesktopGL

### ISSUE THREAD

**Issue by polsys** [MonoGame/MonoGame#5425]
**Steps to reproduce:**
1. Install MonoGame for Windows using the (as of now latest) 3.6.0.1379 installer.
1. Create a new DesktopGL project using the Visual Studio template.
1. Add a new sprite font to the content project. No need to modify the file.
1. Load the font in `Game1`:
```cs
protected override void LoadContent()
{
    spriteBatch = new SpriteBatch(GraphicsDevice);
    font = Content.Load<SpriteFont>("Font");
}
```
5. Draw some text using it:
```cs
protected override void Draw(GameTime gameTime)
{
    GraphicsDevice.Clear(Color.CornflowerBlue);
    spriteBatch.Begin();
    spriteBatch.DrawString(font, "Hello world", new Vector2(16, 16), Color.White);
    spriteBatch.End();

    base.Draw(gameTime);
}
```

No text appears. However, using the 3.6.0.1365 installer and later framework DLL builds, I narrowed it down to 1365 being the last working version and 1366 the first failing. This build contains #5121 and #5139, so either has caused the regression (could be related to @mrhelmut's comment in #5121?).

**Comment by polsys** [comment]
On a further note, definitely related to compressed textures. Setting the font texture format to `Color` solves the issue.

**Comment by Jjagg** [comment]
Thanks for the heads up. I'll see if I can fix this.

**Comment by Jjagg** [comment]
I figured out the issue mrhelmut ran into and will set up a fix in a bit. I'm pretty sure it's not related to this issue though.

**Comment by Jjagg** [comment]
So I thought it wasn't the same issue because we had wrong bindings so it should crash the applications, but I can't reproduce in my branch with the fix #5426

**Comment by mrhelmut** [comment]
The issue in #5121 triggers a crash upon ```Content.Load```, hence if compressed fonts don't crash but don't render, the problem may be elsewhere. I'll try to test #5426 today and see if compressed fonts work.
